db_schema/queries/v1/episodesLTR.sql

```diff
@@ -16,42 +16,33 @@ spotify as (
   AND spotifyEpisodePerformance.account_id = @podcast_id
 ),
 apple as (
-  SELECT 
-  aed_date,
-  ep_name as raw_name,
-  aed_quarter1_median_listeners/aed_histogram_max_listeners*100 as quarter1,
-  aed_quarter2_median_listeners/aed_histogram_max_listeners*100 as quarter2,
-  aed_quarter3_median_listeners/aed_histogram_max_listeners*100 as quarter3,
-  aed_quarter4_median_listeners/aed_histogram_max_listeners*100 as quarter4,
-  aed_histogram_max_listeners as listeners,
-  ep_guid as guid
-  FROM appleEpisodeDetails
-  LEFT JOIN appleEpisodeMetadata USING (episode_id)
+  SELECT * 
+  FROM appleEpisodesLTR
```

Comment:
can't find this view `appleEpisodesLTR`

Reply:
found it and added it in [#e5b376048cef2bf9afa2a8bbf2033f2f2bf4bc6](https://github.com/openpodcast/api/pull/109/commits/e5b376048cef2bf9afa2a8bbf2033f2f2bf4bc67)